From 23b6765105014df8b57ca23891af8049f68760c4 Mon Sep 17 00:00:00 2001
From: "LAPTOP-6H8AMLRC\\ASUS" <lau.cube98@yahoo.com>
Date: Sun, 15 May 2022 18:55:02 +0200
Subject: [PATCH]  rpiPublisher

---
 DeviceConnectorAndSensors/rpiPublisher.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DeviceConnectorAndSensors/rpiPublisher.py b/DeviceConnectorAndSensors/rpiPublisher.py
index 158912a..905b5f5 100644
--- a/DeviceConnectorAndSensors/rpiPublisher.py
+++ b/DeviceConnectorAndSensors/rpiPublisher.py
@@ -18,7 +18,7 @@ POLLING_PERIOD_HR = 1               # chiedo una misurazione ogni 5 minuti
 POLLING_PERIOD_PRESSURE = 2         # chiedo una misurazione ogni 10 minuti
 POLLING_PERIOD_GLYCEMIA = 3        # chiedo una misurazione ogni 20 minuti
 
-ONE_MINUTE_IN_SEC = 60               # per motivi di debug a volte lo metto ad 1 ma deve essere 60
+ONE_MINUTE_IN_SEC = 0               # per motivi di debug a volte lo metto ad 1 ma deve essere 60
                                     # ai fini della dimostrazione potrebbe essere troppo alto e potremmo decidere di abbassarlo
 
 class rpiPub():
-- 
2.31.1.windows.1

